.changeset/pink-melons-rest.md

```diff
@@ -0,0 +1,5 @@
+---
+"@paprika/collapsible": patch
+---
+
+refactor styles to follow latest Paprika conventions for styled-components
```

Comment:
I might consider this a `minor` update... definitely we should include the fact that classnames internal elements have been removed and `pka-anchors` should be used instead – this could break automation tests or styling overrides.